### ISSUE TITLE
Order of Tab Contents Defined in Template Not Preserved

### ISSUE THREAD

**Issue by mcwagner** [BinarCode/vue-form-wizard#315]
It appears when you add the component to modal that it gets re-rendered upon additional clicks of displaying the modal. The generated IDs change so that the order of the tabs and list elements are not retained as defined in the template. Is there a workaround for this occurrence? 

On first open:
![image](https://user-images.githubusercontent.com/5747472/64557243-1ca42380-d30f-11e9-9b87-d2ceae297f2a.png)

After closing and re-opening:
![image](https://user-images.githubusercontent.com/5747472/64557137-eebedf00-d30e-11e9-9b63-b3c7305004b6.png)

**Comment by MrCrankHank** [comment]
Same problem here.

**Comment by wendellpbarreto** [comment]
I'm having this issue too

**Comment by ChrisDevriese** [comment]
Same issue when using one of the examples ... nice module.. can't be used like this.

**Comment by danwalker-caci** [comment]
So far I have had success using vue's keep-alive. I do not reset the component and it maintains the order if I close and re-open wizard.

**Comment by danwalker-caci** [comment]
So the keep-alive appeared to work a few times and now it's not. Makes it less desirable to use.

**Comment by BRafols** [comment]
I don't think that has to do with our library. Does the modal you are using get hidden or recreated when you open/close it?

**Comment by tutuca** [comment]
I'm having this problem without a modal:
```
<form-wizard>
    <tab-content title="1 Customize">
       ...
    </tab-content>
    <tab-content title="2 Schedule">
     ...
    </tab-content>
    <tab-content title="3 Preview">
     ...
    </tab-content>
</form-wizard>
```
gets rendered like this:


![Screenshot from 2022-01-11 14-29-59](https://user-images.githubusercontent.com/70032/148992609-a3cf86b7-8c04-4ab6-b60e-1d1cec90ae79.png)
